tests/release_notes/record/factory/test_default_record_factory.py

```
@@ -332,35 +336,35 @@ def test_generate_with_no_commits_with_wrong_issue_number_in_pull_body_mention(m
 
 def mock_safe_call_decorator_no_issues(_rate_limiter):
     def wrapper(fn):
-        if fn.__name__ == "get_issues_for_pr":
+        if getattr(fn, "__name__", None) == "get_issues_for_pr":
             return mock_get_issues_for_pr_no_issues
         return fn
     return wrapper
 
-def mock_get_issues_for_pr_no_issues(pull_number: int) -> list[int]:
+def mock_get_issues_for_pr_no_issues(pull_number: str) -> list[str]:
```

Ruff (0.13.1)

Unused function argument: `pull_number`

(ARG001)